end_to_end_tests/docstrings-on-attributes-golden-record/my_test_api_client/models/model_with_description.py

```diff
@@ -43,8 +44,8 @@ def to_dict(self) -> dict[str, Any]:
         return field_dict
 
     @classmethod
-    def from_dict(cls: type[T], src_dict: dict[str, Any]) -> T:
-        d = src_dict.copy()
+    def from_dict(cls: type[T], src_dict: Mapping[str, Any]) -> T:
+        d = dict(src_dict)
```

Comment:
The generic `Mapping` type doesn't have `.copy()` or `.pop()`, so this is the best I could think of for making a shallow copy of the input dictionary.

Reply:
I think the code should be equivalent when used on a dict, so that makes sense to me!